README.md

```diff
@@ -1,33 +1,15 @@
-# JS Library
-
-[![Build Status](https://travis-ci.org/SecureTrading/js-payments.svg?branch=develop)](https://travis-ci.org/SecureTrading/js-payments) <!-- Coveralls --> [![Coverage Status](https://coveralls.io/repos/github/SecureTrading/js-payments/badge.svg?branch=develop)](https://coveralls.io/github/SecureTrading/js-payments?branch=develop) <!-- Browserstack --> [![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=eVhLS2ZRY1hqcnJhQXZYVVVZeGhBZnNZL2U4S0ZuNzFDSVk5WFd6MnQvTT0tLVZXd0diVnBPU1RURXJ4S2VrakpuUlE9PQ==--8df058f3f39c1ace5b4d83d27b8c3b46fc3bd90d)](https://www.browserstack.com/automate/public-build/eVhLS2ZRY1hqcnJhQXZYVVVZeGhBZnNZL2U4S0ZuNzFDSVk5WFd6MnQvTT0tLVZXd0diVnBPU1RURXJ4S2VrakpuUlE9PQ==--8df058f3f39c1ace5b4d83d27b8c3b46fc3bd90d) <!-- Dependabot --> [![Dependabot badge](https://img.shields.io/badge/Dependabot-enabled-brightgreen.svg)](https://app.dependabot.com/) <!-- Snyk --> [![Known Vulnerabilities](https://snyk.io/test/github/SecureTrading/js-payments/develop/badge.svg)](https://snyk.io/test/github/SecureTrading/js-payments/develop)
-
-A JavaScript interface for allowing tokenization and authorisation of payments through SecureTrading.
+# JS Payment Card
 
 ## Quickstart
 
 You can check JS Library in action by running it on your local environment. To do this, run commands below:
 
 ```
-npm install
-npm start
-```
-
-Then open address `https://localhost:8443` in your web browser.
-
-Please be aware, that NPM uses the configuration from `package.json` file and by default hostname is set to `localhost`. To use another hostname you should set a configuration variable by running the command below:
-
-```
-npm config set js-payments:host YOUR_HOSTNAME
+yarn
```

Comment:
I thought we were changing back to npm instead of yarn as it wasn't working well?

Reply:
yup, I forgot to change it in Readme

Reply:
changed from `yarn` to `npm`